R/printnum.R

```diff
@@ -16,9 +16,9 @@
 #' @param zero_string Character. Word to print if \code{x} is a zero integer.
 #' @inheritDotParams base::formatC -x
 #'
-#' @details If \code{x} is a vector, \code{digits}, \code{gt1}, and \code{zero} can be vectors
-#'    according to which each element of the vector is formated. Parameters are recycled if length of \code{x}
-#'    exceeds length of the parameter vectors. If \code{x} is a matrix, the vectors specify the formating
+#' @details If \code{x} is a vector, all arguments can be vectors
+#'    according to which each element of the vector is formatted. Parameters are recycled if length of \code{x}
+#'    exceeds length of the parameter vectors. If \code{x} is a matrix or data.frame, the vectors specify the formating
```

Comment:
Could you add code tags to "matrix" and "data.frame" and add a second "t" to "formating" while you are at it?

Reply:
Done.